docs/guides/prefect-deploy.md

```diff
@@ -177,15 +176,16 @@ $ prefect init --recipe docker
 </div>
 
 !!! tip "Use `--field` to avoid the interactive experience"
-    We recommend that you only initialize a recipe when you are first creating your deployment structure, and afterwards store your configuration files within version control. 
+    We recommend that you only initialize a recipe when you are first creating your deployment structure, and afterwards store your configuration files within version control.
     However, sometimes you may need to initialize programmatically and avoid the interactive prompts.  
     To do so, provide all required fields for your recipe using the `--field` flag:
     <div class="terminal">
     ```bash
     $ prefect init --recipe docker \
         --field image_name=my-repo/my-image \
         --field tag=my-tag
-    ```
+
+```
```

Comment:
I think this created a rendering issue

Reply:
Good catches. Closing for now as the auto-fix introduced several bugs you've identified and I suspect some other wonky formatting. Getting rid of the <div class="terminal"> - which now appears reasonable - could help if we decide to go with a markdownlint formatter in the future.